scripts/rebase.sh

```
@@ -565,3 +747,399 @@ function rebase_conflicts {
+            echo -e "5. Abort pull operation: ${YELLOW}abort and reset${ENDCOLOR}"
+            echo -e "0. Exit script"
+            
+            print_menu="false"
+        fi
+        
+        read -n 1 -s choice
+        
+        case "$choice" in
+            1)
+                # Check if conflicts are resolved
+                files_with_conflicts=$(git grep -l --name-only -E "[<=>]{7}" $(git --no-pager diff --name-only --diff-filter=U --relative) 2>/dev/null || echo "")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)